src/static/styles/ui.scss

```diff
@@ -490,8 +490,24 @@
     width: 100%;
     padding: 0;
     border: 0;
-    background: transparent;
+    background-color: transparent;
+
+    &:-webkit-autofill,
```

Comment:
Shouldn't this use the `:autofill` pseudo-class and let postcss or whatever convert this to what is needed for the different browsers?

Reply:
Not yet prefixed by autoprefixer: https://github.com/postcss/autoprefixer/issues/626